src/include/miopen/solver/implicitgemm_ck_util.hpp

```diff
@@ -36,6 +36,8 @@
 #if MIOPEN_BACKEND_HIP && MIOPEN_USE_COMPOSABLEKERNEL
 #include <ck/utility/data_type.hpp>
 #include <ck/library/tensor_operation_instance/gpu/grouped_convolution_backward_weight.hpp>
+#include <ck/library/tensor_operation_instance/gpu/grouped_convolution_backward_weight_bilinear.hpp>
+#include <ck/library/tensor_operation_instance/gpu/grouped_convolution_backward_weight_scale.hpp>
```

Comment:
Probably fine for now, but some of these common utilities are starting to be implementation specific, and likely should be moved to the specific solvers, or a file that is specific for that group of solvers.

On the other hand, if we are planning on adding split-k support to all solvers eventually, then this might be unified again and this special logic can be removed.

Reply:
Yes. I agree. It is a kind of awkward but we need it because now CK only enable split_k support in the 2d and 3d wrw solvers. If finally we have general split-k support, then we can remove the special logic.